test/topology_experimental_raft/test_topology_ops.py

```diff
@@ -39,3 +44,26 @@ async def test_topology_ops(request, manager: ManagerClient):
     logger.info(f"Decommissioning node {servers[1]}")
     await manager.decommission_node(servers[1].server_id)
     await check_token_ring_and_group0_consistency(manager)
+
+
+@pytest.mark.asyncio
+async def test_blocked_bootstrap(manager: ManagerClient):
```

Comment:
Please write a comment
the information provided in commit message should be included here.

Reply:
Done.